TimesMachine.js

```diff
@@ -64,12 +64,9 @@ function scrape(doc, url) {
 		item.pages = ZU.trimInternal(text(doc, '#page_num_content'))
 			.replace(/ ,/g, ',');
 		
-		if (item.abstractNote) {
-			let place = item.abstractNote.match(/^([A-Z]+)\b/);
```

Comment:
I don't think this is right -- 'place' is the place of publication across item types, not the place the reporter is reporting from (which this returns)